### PR TITLE
WRP-7252: Increase automated tests for TabLayout.

### PR DESCRIPTION
**Pull request by dongsuwon**
Enact-DCO-1.0-Signed-off-by: Dongsu Won (dongsu.won@lgepartner.com)

### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
- Need to add ui-test for TabLayout.
- Need to add new ui-test view for TabLayout.
- Need to add ss-test for TabLayout.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Add ui-test for TabLayout.
- Add new ui-test view for TabLayout.
- Add ss-test for TabLayout.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]:# (Related issues, references)
WRP-7252

### CODE REVIEW

File: tests/ui/specs/TabLayout/WithDisabled/TabLayoutWithDisabled-specs.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const Page = require('../TabLayoutPage');
+
+describe('TabLayout', function () {
+	beforeEach(async function () {
+		await Page.open('WithDisabled');
+	});
+
+	describe('With disabled tab', function () {
```
Comment (sjro) [inline tests/ui/specs/TabLayout/WithDisabled/TabLayoutWithDisabled-specs.js:8]: Isn't this a test related to QWTC-2338?

File: tests/ui/specs/TabLayout/WithDisabled/TabLayoutWithDisabled-specs.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		it('should return spotlight after click on navigation button - [QWTC-2503]', async function () {
```
Comment (sjro) [inline tests/ui/specs/TabLayout/WithDisabled/TabLayoutWithDisabled-specs.js:9]: Isn't this a test related to QWTC-2338?

Comment (dongsuwon) [reply tests/ui/specs/TabLayout/WithDisabled/TabLayoutWithDisabled-specs.js]: It's my mistake. I fixed it.

**Review by sjro — APPROVED** [review]
LGTM

**Pull request merged**